server/src/internalClusterTest/java/org/opensearch/cluster/coordination/UnsafeBootstrapAndDetachCommandIT.java

```diff
@@ -47,31 +46,40 @@
 import org.opensearch.gateway.GatewayMetaState;
 import org.opensearch.gateway.PersistedClusterStateService;
 import org.opensearch.indices.IndicesService;
-import org.opensearch.test.OpenSearchIntegTestCase;
 import org.opensearch.test.InternalTestCluster;
+import org.opensearch.test.OpenSearchIntegTestCase;
 
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Locale;
+import java.util.Objects;
 
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
 import static org.opensearch.action.support.WriteRequest.RefreshPolicy.IMMEDIATE;
 import static org.opensearch.gateway.DanglingIndicesState.AUTO_IMPORT_DANGLING_INDICES_SETTING;
 import static org.opensearch.index.query.QueryBuilders.matchAllQuery;
 import static org.opensearch.indices.recovery.RecoverySettings.INDICES_RECOVERY_MAX_BYTES_PER_SEC_SETTING;
 import static org.opensearch.test.NodeRoles.nonMasterNode;
+import static org.opensearch.test.hamcrest.OpenSearchAssertions.assertAcked;
 import static org.opensearch.test.hamcrest.OpenSearchAssertions.assertHitCount;
-import static org.hamcrest.Matchers.containsString;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.notNullValue;
```

Comment:
revert all the un-necessary import changes due to intellij or check with @saratvemulapalli  which code formatter to use.

Reply:
I guess IntelliJ has reformatted all changes according to project rules only. See below check which is passing.

`
┌[kiwish-4.2]-(workplace/harmishlakhani-OpenSearch)-[git:unsafe_bootstrap_read_only_block]-
└> ./gradlew spotlessApply

> Configure project :
========================= WARNING =========================
         Backwards compatibility tests are disabled!
See https://github.com/opensearch-project/OpenSearch/issues/105
===========================================================

> Configure project :qa:os
Cannot add task 'destructiveDistroTest.docker' as a task with that name already exists.
=======================================
OpenSearch Build Hamster says Hello!
  Gradle Version        : 6.6.1
  OS Info               : Mac OS X 10.15.7 (x86_64)
  Runtime JDK Version   : 11 (JDK)
  Runtime java.home     : /Library/Java/JavaVirtualMachines/amazon-corretto-11.jdk/Contents/Home
  Gradle JDK Version    : 15 (JDK)
  Gradle java.home      : /Library/Java/JavaVirtualMachines/amazon-corretto-15.jdk/Contents/Home
  Random Testing Seed   : 6884CF4EAA1728F3
  In FIPS 140 mode      : false
=======================================

BUILD SUCCESSFUL in 2s
23 actionable tasks: 23 up-to-date
┌[kiwish-4.2]-(workplace/harmishlakhani-OpenSearch)-[git:unsafe_bootstrap_read_only_block]-
└> git status
On branch unsafe_bootstrap_read_only_block
Your branch is up to date with 'origin/unsafe_bootstrap_read_only_block'.

nothing to commit, working tree clean
`